### PR TITLE
User profile fixes

### PR DESCRIPTION
**Pull request by CptTony**
CUR-433
CUR-453
CUR-461
CUR-470
CUR-476

### CODE REVIEW

File: src/containers/Account/ProfilePage.js [hunk 1]
```diff
@@ -200,14 +205,14 @@ function ProfilePage(props) {
...
                       <div className="form-group">
                         <label htmlFor="job-title">Job Title</label>
-                        <FontAwesomeIcon icon="user" />
+                        <FontAwesomeIcon icon="briefcase" />
                         <input
                           className="input-box"
                           type="text"
                           id="job-title"
                           name="jobTitle"
                           placeholder="Job Title*"
```
Comment (simonebelotti75) [inline src/containers/Account/ProfilePage.js:214]: Remove * here

File: src/containers/Account/ProfilePage.js [hunk 2]
```diff
@@ -242,28 +231,33 @@ function ProfilePage(props) {
...
+                            type="submit"
+                            className="btn btn-primary submit"
+                            disabled={isLoading}
+                          >
+                            {isLoading ? (
+                              <img src={loader} alt="" />
+                            ) : (
+                              'Update Profile'
+                            )}
+                        </button>
```
Comment (simonebelotti75) [inline src/containers/Account/ProfilePage.js:253]: Indent issues

File: src/containers/Account/ProfilePage.js [hunk 3]
```diff
@@ -242,28 +231,33 @@ function ProfilePage(props) {
...
+                            ) : (
+                              'Update Profile'
+                            )}
+                        </button> 
+                    </div>
+                  </div>
+                  <div className="row">
+                    <div className="col">
+                      <Error error={error} />
+                    </div>
```
Comment (simonebelotti75) [inline src/containers/Account/ProfilePage.js:259]: Do we need wrap this?

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/containers/Account/ProfilePage.js (hunk 3 above)
Comment (CptTony) [reply src/containers/Account/ProfilePage.js:259]: It helps keep consistent spacing and padding in bootstrap.

Not a hard requirement obviously but I'm gonna say yeah.

**Review by simonebelotti75 — APPROVED** [review]

**Pull request merged**